data/json/items/armor/helmets.json

```diff
@@ -77,10 +77,10 @@
     "category": "armor",
     "name": { "str": "tactical full helmet" },
     "description": "An all-encompassing black helmet that covers your entire face and neck, providing excellent protection from all sorts of damage.",
-    "weight": "1324 g",
-    "volume": "2 L",
-    "price": 50000,
-    "price_postapoc": 1000,
+    "weight": "1594 g",
+    "volume": "2250 ml",
+    "price": 25000,
+    "price_postapoc": 2500,
```

Comment:
I guess this is wrong. Sturdy helmets should be one of the first things to seek after the apocalypse, and thus they shouldn't cost mere $25. I'd set the `price_postapoc` to at least of its pre-apocalypse value, or maybe even raise it a bit.

Reply:
also while you're at it, make it a string :pray:

Reply:
Strange, the army helmet was listed as 1500 postapoc.

Reply:
I've looked through the other items, and it seems that the postapoc pricing is relative. For example, the A7 Laser rifle, a more endgame item is 7500 postapoc, or 75$.

Reply:
> 
> 
> also while you're at it, make it a string 🙏

what do you mean "make it a string"

Reply:
We need a total overhaul of postapoc prices, so we can start with this helmet.